src/mediapipe_internal/mediapipegraphdefinition.hpp

```diff
@@ -93,7 +93,7 @@ class MediapipeGraphDefinition {
     static constexpr model_version_t VERSION = 1;
 
 protected:
-    std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> pythonNodeResources;
+    std::unordered_map<std::string, std::shared_ptr<PythonNodeResources>> pythonNodeResources;
```

Comment:
Are we using shared_ptr here so that ongoing executions could use old resources while the graph can be reloaded?

Reply:
Yes, as far as I know if the gRPC stream is on and we unload the graph it will keep working until the stream is done even though we clean up resources map and shared_ptr here is enabling extending the lifetime.